awx/settings/local_settings.py.docker_compose

```diff
@@ -108,40 +64,6 @@ PROXY_IP_WHITELIST = []
 # Enable logging to syslog. Setting level to ERROR captures 500 errors,
 # WARNING also logs 4xx responses.
 
-LOGGING['handlers']['syslog'] = {
-    'level': 'WARNING',
-    'filters': ['require_debug_false'],
-    'class': 'logging.NullHandler',
-    'formatter': 'simple',
-}
-
-LOGGING['loggers']['django.request']['handlers'] = ['console']
```

Comment:
This part _may_ be controversial. From what I could tell, this doesn't change what goes to the console in the dev env. It just stops logs from going to files.

Way back in the day, church used to do his devel outside of a container or even maybe a VM. So it was important to not litter your host machine with log files.

Today, all dev takes place in containers. The log files are rotated so no risk of eating up all the disk space either.

Reply:
given your last point I'm not sure this is actually controversial.

Reply:
but then are unit tests and stuff like that going to dump a whole bunch of garbage to these log files?

Reply:
This is also done here:

https://github.com/ansible/awx/blob/abc6a842105999bfaadf312ed1fb9596574355bf/installer/roles/image_build/files/settings.py#L55

Sharing code between development and this seems like a good idea, but I don't like the idea of removing this in the name of simplicity. Logging to the console is the doctrine of Kubernetes